Classes/Domain/Model/Index.php

```diff
@@ -298,4 +298,13 @@ private function prefixName(): string
 
         return $indexConfiguration['prefix'] . '-' . $this->name;
     }
+
+    private function encodeRequestBody(array $content): string
+    {
+        if (empty($content)) {
```

Comment:
`$content === []`

Reply:
I find empty more "speaking" :)

Reply:
Fine… for me it's "fuzzy" 🙃

Rather non-scientific, but `empty` uses 10ms, the strict check 9ms on 3v4l.org. Which is 10% (if true). 😎